.palantir/revapi.yml

```diff
@@ -489,6 +489,9 @@ acceptedBreaks:
         \ @ org.apache.iceberg.SnapshotScan<ThisT, T extends org.apache.iceberg.ScanTask,\
         \ G extends org.apache.iceberg.ScanTaskGroup<T extends org.apache.iceberg.ScanTask>>"
       justification: "Removing deprecations for 1.3.0"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.TableOperations org.apache.iceberg.metrics.CommitReport::tableOperations()"
```

Comment:
as I mentioned further below, `TableOperations` shouldn't be added to `CommitReport`. A `CommitReport` represents some final results from a commit that happened